backend/utils/cache.py

```diff
@@ -0,0 +1,5 @@
+MINUTE_IN_SECONDS = 60
```

Comment:
I think maybe an `IntEnum` might work better here? Would be easier to import to other files and also syntax-wise we can do `Cache.SECONDS` or like `Cache.MINUTE` or `30 * Cache.SECONDS`

Reply:
sgtm!